api/graphql/schema.graphql

```diff
@@ -721,7 +721,13 @@ input CreateUserInput {
     # Personal info
     firstName: String! @rename(attribute: "first_name")
     lastName: String! @rename(attribute: "last_name")
-    email: Email @rules(apply: ["unique:users,email"])
+    email: Email @rules(apply: ["unique:users,email"],
+    messages: [
+        {
+            rule: "unique"
+            message: "Email address already exists"
+        }
+    ])
```

Comment:
Even though it's just a spike, to side-step confusion for anyone else reviewing :)

```suggestion
    email: Email @rules(
        apply: ["unique:users,email"],
        messages: [{
            rule: "unique"
            message: "Email address already exists"
        }]
    )
```